firestore-translate-text/functions/src/index.ts

```diff
@@ -32,16 +34,17 @@ enum ChangeType {
   DELETE,
   UPDATE,
 }
-
+const DOCS_PER_BACKFILL = 200;
```

Comment:
Did we measure how long it takes to translate 200 docs?

Reply:
I ran some testing on this amount of records, which only took 3 seconds.

There seems to be a delay on the initial processing status, with task queueing.

![image](https://user-images.githubusercontent.com/2060661/202852237-099781ea-8d3b-4285-8238-7d2a6dfc1707.png)

It will fail once, retry and then process the documents.

Reply:
Yes, it's normal for the task to fail the first few times. There's nothing we can do about this. It's because we've just created the queue, and the function, and granted function runner permissions, and those permissions take a bit to propagate. As long as the queue has default or better retries, the function will run as soon as the permissions have propagated.

Reply:
That's reasonable. We should stay pretty conservative tho, because real documents can have many fields, each of which need to be translated individually into many languages. I'm going to increase this to 1500, which should be very safe